rust/src/dns/log.rs

```diff
@@ -86,6 +86,7 @@ pub const LOG_URI        : u64 = BIT_U64!(59);
 
 pub const LOG_FORMAT_GROUPED  : u64 = BIT_U64!(60);
 pub const LOG_FORMAT_DETAILED : u64 = BIT_U64!(61);
+pub const LOG_HTTPS      : u64 = BIT_U64!(62);
```

Comment:
Let's try to keep it the same number of spaces before `:` as above.

Reply:
Alright, I kept this based on the spacing on lines above line 87 due to its length. I will do this after completing the to do's

Reply:
I certainly did not check that. This seems to require some formatting. Let's leave it where it is as of now. Sorry about the trouble. 😅